app/exporters/formatters/utaac_decision_indexable_formatter.rb

```diff
@@ -9,13 +9,13 @@ def type
   def extra_attributes
     {
       indexable_content: "#{entity.hidden_indexable_content}\n#{entity.body}".strip,
-      tribunal_decision_category: entity.tribunal_decision_category,
-      tribunal_decision_category_name: expand_value(:tribunal_decision_category).first,
+      tribunal_decision_categories: entity.tribunal_decision_categories,
+      tribunal_decision_categories_name: expand_value(:tribunal_decision_categories),
```

Comment:
I think it makes more sense to call this attribute `tribunal_decision_category_names `.
What do you think?